### DISCUSSION TITLE
show and hide with inline mode?

### DISCUSSION THREAD

**Discussion by hajo-travanto** [easepick/easepick#55] [Q&A] [answered]
hi, great work with the easepick project. thank you so much for it!

at big and medium display resolutions its working fine, but for (width < 450) a different easepick setup is required with "inline: true" since the easepick rangepicker won't show anymore. with that inline-setting the show and hide methods are not working.

i already tried a lot to work around this issue, but except for hacking the display-property of the "easepick-wrapper" span i didn't find a solution and this created some additional complexity in the code that i would like to remove sooner or later.

any ideas or suggestions? :)

**Comment by wakirin** [comment]
Could you provide a link to see the issue?

**Comment by hajo-travanto** [comment, marked as answer]
i solved it now removing the "bottom:0!important;" from ".container:not(.inline).show" inside the css. with that change i was able to use the same easepick for all resolutions without using "inline:true". if show/hide should have worked with inline-mode i could try to create a code sample.

**Reply by hajo-travanto** [reply]
i checked the hide/show with inline:false and it is not working, too, while isShown works perfectly showing "true" when it is there and "false" when it is not, the hide/show methods have no impact on it. i am using the core with the lock- and range-plugin. i'll try to add a code example later that can reproduce this, because it works with a basic easepick setup.

**Reply by hajo-travanto** [reply]
can't reproduce the issue in a simple way, sorry. i wonder a bit why show() requires an event argument in typescript while hide() does not. funny thing is that many other methods like destroy() are working fine.
